sdk/python/responsible-ai/responsibleaidashboard-diabetes-decision-making/responsibleaidashboard-diabetes-decision-making.ipynb

```diff
@@ -333,10 +333,15 @@
     "    X_train = all_data.drop(labels=args.target_column_name, axis=\"columns\")\n",
     "    print(\"X_train cols: {0}\".format(X_train.columns))\n",
     "\n",
+    "    # Declare any features that you don't want to train in the model\n",
```

Comment:
is this dropped feature meant to be in this PR? didn't see it in the PR description, wanna make sure...

Reply:
this was checked in in vNext repo and missed here in azureml-assets. Added in the description!